compiler/rustc_parse/src/validate_attr.rs

```diff
@@ -84,19 +84,24 @@ pub fn parse_meta<'a>(sess: &'a ParseSess, attr: &Attribute) -> PResult<'a, Meta
     })
 }
 
-pub fn check_meta_bad_delim(sess: &ParseSess, span: DelimSpan, delim: MacDelimiter, msg: &str) {
+pub fn check_meta_bad_delim(sess: &ParseSess, span: DelimSpan, delim: MacDelimiter) {
```

Comment:
Split these to two diagnostics because of https://github.com/projectfluent/fluent/wiki/Good-Practices-for-Developers#prefer-wet-over-dry. I'm not entirely sure that there really needs to be a special message for `cfg_attr` however